### ISSUE TITLE
Accessibility: Content reading order does not follow meaningful order [Add to Cart]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52142]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

When the setting to redirect to the cart page after add to cart is turned on, there is a status message that appears at the top of the cart page after each add to cart action. When listening to this status message with a screen reader, the content is heard out of order or read in a way that is confusing or lacks meaning.   

Illogical reading order is being observed:
1.Continue Shopping button >> 2. Status message “Aerodynamic Steel Table” has been added to your cart.

![Image](https://github.com/user-attachments/assets/59dd0baf-6a3b-4fe2-ae38-75e7f0314116)

### Component or Page: 

Add to cart redirect setting toggled on

### Name affected element:

Status message “Product Name” has been added to your cart and Continue Shopping button on cart page after adding product to cart

### Relevant code snippets:

`<div class=""wc-block-components-notice-banner__content"">
			<a href=""https://woo.equalizedigital.com/?product=aerodynamic-steel-table"" tabindex=""1"" class=""button wc-forward wp-element-button"">Continue shopping</a> “Aerodynamic Steel Table” has been added to your cart.		</div>`

### Recommended fix:

After redirecting to the cart page, ensure that the first thing screen readers read out is the success message.

Change the DOM order so the continue shopping button ocurrs after the status message (as it visually appears).

**Comment by Manussakis** [comment]
This issue was fixed when the https://github.com/woocommerce/woocommerce/pull/50061 PR was merged.

cc: @vikrampm1
